src/main/java/edu/rosehulman/quota/Tier.java

```diff
@@ -5,7 +5,20 @@
   private String name;
   private int max; // TODO: This might need to be changed to BigInt or BigDouble
   private int value;
-  private double price; // TODO: Not a good idea to save price in a double format
+  private double price; // TODO: Not a good idea to save price in a double
+                        // format
```

Comment:
We should have everyone get rid of the line length limit in their ide settings. 

https://stackoverflow.com/questions/3697287/eclipse-set-maximum-line-length-for-auto-formatting